### PR TITLE
Added initial content_tag and input_tag

### PR DESCRIPTION
**Pull request by skunkworker**
With text_field_tag, label_tag and content_tag with block supported. This is still very early WIP, but I tried to make it follow the rails *_tag syntax as closely as possible.

### CODE REVIEW

File: src/jasper/helpers/forms.cr [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module Jasper::Helpers::Forms
+
+  def text_field_tag(**options)
```
Comment (elorest) [inline src/jasper/helpers/forms.cr]:
```cr
def text_field_tag(**options : Object)
end
``` 
Will stop it from responding to empty values unless you want that.

File: src/jasper/helpers/forms.cr [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    options = options.to_h
+
+    input_field_tag(type: :text, options: options)
+  end
+
+  def label_tag(**options)
```
Comment (elorest) [inline src/jasper/helpers/forms.cr]: You could probably clean this up a bit with this overload.

```cr
  def label_tag(content : Symbol, **options : Object)
    options = options.to_h
    content_tag(name: :label, content: content, options: options)
  end
```

Comment (skunkworker) [inline src/jasper/helpers/forms.cr]: I've updated this.

File: src/jasper/helpers/forms.cr (hunk 1 above)
Comment (skunkworker) [inline src/jasper/helpers/forms.cr]: Hmm is there a use case for no options?

**Comment by skunkworker** [comment]
We need to also decide how to do form objects and have them interface with the orm.

**Comment by eliasjpr** [comment]
Issue https://github.com/Amber-Crystal/jasper_helpers/issues/5

**Pull request merged**